### PR TITLE
Change defaultMessage for spoiler placeholder

### PR DESCRIPTION
**Pull request by ykzts**
follow up #4313

### CODE REVIEW

File: app/javascript/mastodon/locales/defaultMessages.json [hunk 1]
```diff
@@ -968,27 +968,6 @@
...
-        "defaultMessage": "Submit",
-        "id": "report.submit"
-      },
-      {
-        "defaultMessage": "Reporting",
-        "id": "report.target"
-      }
-    ],
-    "path": "app/javascript/mastodon/features/report/index.json"
-  },
```
Comment (ykzts) [inline app/javascript/mastodon/locales/defaultMessages.json]: https://github.com/tootsuite/mastodon/commits/master/app/javascript/mastodon/features/report/index.js has been deleted?

Comment (Gargron) [inline app/javascript/mastodon/locales/defaultMessages.json]: That file has been deleted, but the keys remain in the report modal. I cannot reproduce the deletion of all these report keys when I run yarn run manage:translations on master.

**Review by Gargron — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/locales/defaultMessages.json (hunk 1 above)
Comment (ykzts) [inline app/javascript/mastodon/locales/defaultMessages.json]: What happens if I run `build:development` after deleting the build directory once?

```console
$ rm -rf build
$ yarn build:development
$ yarn manage:translations
```

Comment (Gargron) [inline app/javascript/mastodon/locales/defaultMessages.json]: Okay, can confirm. My bad.

**Review by Gargron — APPROVED** [review]

**Pull request merged**